test/addons/csi-addons/kustomization.yaml

```diff
@@ -8,5 +8,5 @@ resources:
   - https://github.com/csi-addons/kubernetes-csi-addons/releases/download/v0.10.0/setup-controller.yaml
 
 images:
-  - name: quay.io/csiaddons/k8s-controller
-    newTag: v0.10.0
+  - name: quay.io/ypadia/k8s-controller
```

Comment:
why `ypadia`?

Reply:
Because I want to test this PR now:
https://github.com/csi-addons/kubernetes-csi-addons/pull/691

Reply:
image kustomization fixed, it works now:
https://github.com/csi-addons/kubernetes-csi-addons/pull/691#issuecomment-2457238528